engine/src/main/java/org/opencds/cqf/cql/engine/elm/execution/EquivalentEvaluator.java

```diff
@@ -92,14 +92,22 @@ else if (left instanceof String && right instanceof String) {
             return ((String) left).equalsIgnoreCase((String) right);
         }
 
-        return Context.getContext().objectEquivalent(left, right);
+        if (context != null) {
+            return context.objectEquivalent(left, right);
+        }
+
+        return null;
```

Comment:
Same issue, if the default behavior can't be determined and we need to ask a data provider to perform the equivalent comparison, then it is an error if there is no context.